### PR TITLE
Implement new user menu - Closes #4468

### LINKED ISSUE #4468: Create user menu
### Description
User needs to navigate to the pages for manage account such as Switch account, Backup account, Add new account, Upgrade to multisignature, Edit name and Remove Account. So after user selects an account, the Lisk logo will be replaced with the user menu

### Acceptance Criteria
- Replace Lisk logo with user avatar, address and name
- Ensure [UI/UX design](https://www.figma.com/file/9JbvSmtLNJIttXp68uhmQR/Change-API-to-support-Lisk-SDK-v6-API?node-id=836%3A87932) requirements are met
- Add required unit tests. 
- Add required e2e tests.

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #4468

### How was it solved?

Updating the UI to the new [UI design](https://www.figma.com/file/9JbvSmtLNJIttXp68uhmQR/Change-API-to-support-Lisk-SDK-v6-API?node-id=836%3A87932)

### How was it tested?

Visually

### CODE REVIEW

**Review by Balanced02 — DISMISSED** [review]
LGTM!

**Review by Balanced02 — DISMISSED** [review]

File: src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    color: var(--color-blue-gray);
+  }
+}
+
+.dropDownMenu {
+  border: 0;
+  min-width: 180px !important;
+  top: calc(100% + 3px) !important;
+  margin-left: -30px;
+  background-color: var(--color-background-dropdown);
```
Comment (soroushm) [inline src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.css]: background color dosn't match with design
color : #030815

File: src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            <span className={styles.name}>{name}</span>
+            <span className={styles.address}>{truncateAddress(address)}</span>
+          </div>
+          <Icon name="dropdownArrowIcon" />
+        </>
+      }
+      size="m"
+    >
+      <ul className={styles.dropDownMenuList}>
+        {ACCOUNT_MENU.map(({ path, icon, label, component }) => (
```
Comment (soroushm) [inline src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.js:35]: remove account should be red

**Review by soroushm — CHANGES_REQUESTED** [review]

File: src/modules/account/const/accountMenu.js [hunk 3]
```diff
@@ -17,6 +17,16 @@ export const ACCOUNT_MENU = [
     icon: 'refresh',
     label: 'Backup account',
   },
+  {
+    path: routes.addAccount.path,
+    icon: 'plus',
+    label: 'Add new account',
+  },
+  {
+    path: modals.multiSignature.path,
```
Comment (soroushm) [inline src/modules/account/const/accountMenu.js]:
```suggestion
    path: `/${route.delegates.path}?modal=${modals.multiSignature.path}`,
```

**Review by soroushm — CHANGES_REQUESTED** [review]

File: src/modules/account/const/accountMenu.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/account/const/accountMenu.js]: @soroushm Fixed

File: src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.js (hunk 2 above)
Comment (ikem-legend) [reply src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.js:35]: @soroushm Done

File: src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.css (hunk 1 above)
Comment (ikem-legend) [reply src/modules/account/components/AccountManagementDropdown/AccountManagementDropdown.css]: @soroushm Fixed

**Review by Balanced02 — DISMISSED** [review]

File: src/theme/dialog/holder.js [hunk 4]
```diff
@@ -18,8 +17,10 @@ const DialogHolder = ({ history }) => {
   }, [history.location.search]);
 
   const activeToken = useSelector(selectActiveToken);
-  const networkIsSet = useSelector(state => !!state.network.name);
-  const account = useSelector(selectActiveTokenAccount);
+  const networkIsSet = useSelector((state) => !!state.network.name);
+  const wallet = useSelector(selectActiveTokenAccount);
+  const [account] = useCurrentAccount();
+  const isAuthenticated = Object.keys(account).length > 0 || !!wallet.summary;
```
Comment (soroushm) [inline src/theme/dialog/holder.js]: selectActiveTokenAccount is deprecated now
```suggestion
  const [account] = useCurrentAccount();
  const isAuthenticated = Object.keys(account).length > 0;
```

**Comment by soroushm** [comment]
@ikem-legend background only will change  while menu is open
https://www.figma.com/file/9JbvSmtLNJIttXp68uhmQR/Change-API-to-support-Lisk-SDK-v6-API?node-id=492%3A47457

**Review by soroushm — APPROVED** [review]

**Review by Balanced02 — APPROVED** [review]

**Pull request merged**